<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordSet</class>
 <widget class="QWidget" name="PasswordSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>310</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget{background-image: url(:/new/imgs/images/password (1).png);}</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_old">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_new1">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>150</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_new2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>200</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_save">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>250</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../../imgs.qrc">
      <normaloff>:/new/imgs/images/save.png</normaloff>:/new/imgs/images/save.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>90</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../imgs.qrc"/>
 </resources>
 <connections/>
</ui>
